docs/demos/dev-select/view.html

```diff
@@ -1,5 +1,15 @@
 <div ng-controller="DevSelectCtrl">
-  <a href="#" editable-select="user.status" e-multiple e-ng-options="s.value as s.text for s in statuses">
```

Comment:
Thanks for the work @ckosloski. When will this be merged I could really do with this functionality?